docs/master/digging-deeper/feature-toggles.md

```diff
@@ -25,9 +25,46 @@ type Query {
 }
 ```
 
+## @feature
+
+The [@feature](../api-reference/directives.md#feature) directive allows to include fields in the schema depending
+on a [Laravel Pennant](https://laravel.com/docs/pennant) feature.
+
+For example, you might want a new experimental field only to be available when the according feature is active:
+
+```graphql
+type Query {
+  experimentalField: String! @feature(name: "new-api")
+}
+```
+
+In this case, `experimentalField` will only be included when the `new-api` feature is active. 
+
+Another example would be to only include a field when the feature is inactive:
+
+```graphql
+type Query {
+  deprecatedField: String! @feature(name: "new-api", when: "INACTIVE")
```

Comment:
```suggestion
  deprecatedField: String! @feature(name: "new-api", when: INACTIVE)
```

If not already the case, tests and other examples should also use this as a proper enum.

Reply:
I double-checked, but this was the only typo. Good catch!